### ISSUE TITLE
[Screenshots] Devstream under 'How it Works See It in Action' section

### ISSUE THREAD

**Issue by saurabh100ni** [layer5io/layer5#4887]
### Current Behavior
Currently, [every integration](https://layer5.io/cloud-native-management/meshery/integrations/devstream) has only two old screenshots of the designer and visualizer in the 'How it Works See It in Action' section.

### Desired Behavior
- Add new images and videos from the latest version of the software ([MeshMap](https://layer5.io/cloud-native-management/meshmap))
- Add screenshots for [Devstream](https://layer5.io/cloud-native-management/meshery/integrations/devstream) integration.

### Acceptance Tests
- Ensure your screenshot represents the use of a few specific integrations.
- The screenshot from the Designer mode should contain a valid design, refer to [meshery catalog](https://meshery.io/catalog) to use and view published designs.
- Add both Dark Mode and Light Mode screenshots.
- The screenshot should not be a snippet, take the full screenshot of your browser.
- Use Incognito mode on Chrome browser, to ensure your bookmark bar or any other items are not shown in the screenshot/video.

---

<img src="https://raw.githubusercontent.com/layer5io/layer5/master/.github/assets/images/layer5/5-light-small.svg" width="24px" align="left" /><h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
-  🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

<img src="https://raw.githubusercontent.com/layer5io/layer5/master/.github/assets/images/buttons/community.webp" height="22px" align="left" />Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

**Comment by PRANJALRANA11** [comment]
assign this issue to me

**Comment by ValkonX33** [comment]
hey i would like to work on this @saurabh100ni

**Comment by abhijeetgauravm** [comment]
Since @PRANJALRANA11 is already working on two or more issues. Assigning it to u @ValkonX33

**Comment by abhijeetgauravm** [comment]
Hey @ValkonX33, yeah you need to set up the project locally. Please feel free to post your doubts with screenshots on our [discuss forum](http://discuss.layer5.io/) and please make sure you join our Slack channel too where you can also ask your queries whenever you get stuck in any problem.

**Comment by ValkonX33** [comment]
@saurabh100ni hey can you approve of screenshots that you want in [https://layer5.io/cloud-native-management/meshery/integrations/devstream](every integration) page 

![image](https://github.com/layer5io/layer5/assets/77388132/1d233685-ece0-48ce-a3c1-aba2ec97769b)

**Comment by saurabh100ni** [comment]
@abhijeetgauravm may be able to help with this

**Comment by abhijeetgauravm** [comment]
> @saurabh100ni hey can you approve of screenshots that you want in [https://layer5.io/cloud-native-management/meshery/integrations/devstream](every integration) page
> 
> ![image](https://user-images.githubusercontent.com/77388132/269535488-1d233685-ece0-48ce-a3c1-aba2ec97769b.png)
> 
> still haven't applied Acceptance Test conditions but wanna make sure if you want dashboard in one of them?

hey @ValkonX33 dashboard screenshot isn't required here. You have to add the new screenshots of the designer and visualizer from the latest version of the software ([MeshMap](https://layer5.io/cloud-native-management/meshmap)).

**Comment by ValkonX33** [comment]
> > @saurabh100ni hey can you approve of screenshots that you want in [https://layer5.io/cloud-native-management/meshery/integrations/devstream](every integration) page
> > ![image](https://user-images.githubusercontent.com/77388132/269535488-1d233685-ece0-48ce-a3c1-aba2ec97769b.png)
> > still haven't applied Acceptance Test conditions but wanna make sure if you want dashboard in one of them?
> 
> hey @ValkonX33 dashboard screenshot isn't required here. You have to add the new screenshots of the designer and visualizer from the latest version of the software ([MeshMap](https://layer5.io/cloud-native-management/meshmap)).


just to confirm this is new version of meshmap right? 
i tried to import some of the designs from the meshery catalog but couldn't figure it out on my own is there any specific tutorial on how i can set meshery visualizer and designer using pre-existing designs from catalog so that i can move forward with my issue

**Comment by abhijeetgauravm** [comment]
> > > @saurabh100ni hey can you approve of screenshots that you want in [https://layer5.io/cloud-native-management/meshery/integrations/devstream](every integration) page
> > > ![image](https://user-images.githubusercontent.com/77388132/269535488-1d233685-ece0-48ce-a3c1-aba2ec97769b.png)
> > > still haven't applied Acceptance Test conditions but wanna make sure if you want dashboard in one of them?
> > 
> > 
> > hey @ValkonX33 dashboard screenshot isn't required here. You have to add the new screenshots of the designer and visualizer from the latest version of the software ([MeshMap](https://layer5.io/cloud-native-management/meshmap)).
> 
> just to confirm this is new version of meshmap right? i tried to import some of the designs from the meshery catalog but couldn't figure it out on my own is there any specific tutorial on how i can set meshery visualizer and designer using pre-existing designs from catalog so that i can move forward with my issue

hey @ValkonX33 it seems like u haven't meshmap access. Have u filled the newcomers form and meshmap signup form?

**Comment by ValkonX33** [comment]
> > > > @saurabh100ni hey can you approve of screenshots that you want in [https://layer5.io/cloud-native-management/meshery/integrations/devstream](every integration) page
> > > > ![image](https://user-images.githubusercontent.com/77388132/269535488-1d233685-ece0-48ce-a3c1-aba2ec97769b.png)
> > > > still haven't applied Acceptance Test conditions but wanna make sure if you want dashboard in one of them?
> > > 
> > > 
> > > hey @ValkonX33 dashboard screenshot isn't required here. You have to add the new screenshots of the designer and visualizer from the latest version of the software ([MeshMap](https://layer5.io/cloud-native-management/meshmap)).
> > 
> > 
> > just to confirm this is new version of meshmap right? i tried to import some of the designs from the meshery catalog but couldn't figure it out on my own is there any specific tutorial on how i can set meshery visualizer and designer using pre-existing designs from catalog so that i can move forward with my issue
> 
> hey @ValkonX33 it seems like u haven't meshmap access. Have u filled the newcomers form and meshmap signup form?

yes i did also got the mail but was having issue setting it up so i asked on community but still haven't got the reply

**Comment by saurabh100ni** [comment]
@ValkonX33 you may get access to MeshMap soon,
@Yashsharma1911 I wonder if you can help with access 👀

**Comment by ValkonX33** [comment]
![image](https://github.com/layer5io/layer5/assets/77388132/02d15735-b259-4bbb-adc0-ec5675f486d1)

Hey do you think it'll work as new Visualizer Screenshot ?

**Comment by ValkonX33** [comment]
> > ![image](https://user-images.githubusercontent.com/77388132/270183513-02d15735-b259-4bbb-adc0-ec5675f486d1.png)
> > Hey do you think it'll work as new Visualizer Screenshot ? I'll open it in incognito and remove bookmarks once i confirm i'm on right page @saurabh100ni
> 
> Let's discuss this Item in today's website call, Please add it as your agenda if you would: https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit

ah missed it do i have to wait until next one?

**Comment by saurabh100ni** [comment]
Yes @ValkonX33

**Comment by shivankurchavan** [comment]
id like to work on this issue

**Comment by saurabh100ni** [comment]
any updates @shivankurchavan ?

**Comment by shivankurchavan** [comment]
Hello, @saurabh100ni , apologies for delay. I'll let you know if I face any issue. Will complete this asap

**Comment by sandramsc** [comment]
This issue has been open for some time with no recent activity, unassigning to open it up for new contributors to give it a go.

**Comment by surajvast1** [comment]
I would like to work on this

**Comment by saurabh100ni** [comment]
here you go @surajvast1 🚀

**Comment by saurabh100ni** [comment]
any updates here? @surajvast1

**Comment by surajvast1** [comment]
@saurabh100ni the docker scrrenshot issue is not yet merged and it was commited from my master branch . I want it to be merged first to work on further such issues

**Comment by saurabh100ni** [comment]
It's okay this time @surajvast1, but from next time, make sure you create separate branch while working on seperate issues, okay?

**Comment by surajvast1** [comment]
I get it

**Comment by jagpreetrahi** [comment]
Hey @Rajesh-Nagarajan-11 , I would like to work on this

**Comment by mohammedyehaya2003** [comment]
Hi @saurabh100ni,

I would like to work on this issue as my first open-source contribution.

Could you please assign this issue to me?

Thank you!
